docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,59 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a {doc}`native Azure implementation
+</object-storage/file-system-azure>` that does not rely on the legacy Hadoop
+implementation. Trino recommends upgrading existing deployments to the new
+native implementation at your earliest convenience.
+
+The following considerations apply to legacy configurations moving to the
+native implementation:
+
+* Legacy storage types (WASB and ADL) are not supported.
```

Comment:
This is probably not worth mentioning since it isnt supported by MS and by the legacy Hadoop file system stuff either.